libstuff/SQResult.cpp

```diff
@@ -152,6 +153,10 @@ size_t SQResult::size() const {
     return rows.size();
 }
 
+const vector<SQResultRow>& SQResult::getRows() const {
+    return rows;
+}
```

Comment:
It looks like this is only used in one place in auth, `BillExpensifyOrgDonation::processTransactions`.

I would rather remove this getter, and update `BillExpensifyOrgDonation::processTransactions` to take an SQResult.